quick_tour/the_big_picture.rst

```diff
@@ -29,12 +29,13 @@ When developing a Symfony application, your responsibility as a developer
 is to write the code that maps the user's *request* (e.g. ``http://localhost:8000/``)
 to the *resource* associated with it (the ``Homepage`` HTML page).
 
-The code to execute is defined in **actions** and **controllers**. The mapping
-between user's requests and that code is defined via the **routing** configuration.
-And the contents displayed in the browser are usually rendered using **templates**.
+The code to execute is defined in **controllers**  also called **actions** which
```

Comment:
I think this sounds okej. My words were actually based on the paragraf from Controller chapter which goes like this (http://symfony.com/doc/current/book/controller.html#a-simple-controller):

"Note that the *controller* is the ``indexAction`` method, which lives inside a *controller class* (``HelloController``). Don't be confused by the naming: a *controller class* is simply a convenient way to group several controllers/actions together. Typically, the controller class will house several controllers/actions (e.g. ``updateAction``, ``deleteAction``, etc)."